src/client/yoti.client.js

```diff
@@ -29,13 +29,23 @@ class YotiClient {
   /**
    * @param {string} sdkId
    * @param {string} pem
+   * @param {Object} options
+   * @param {string} options.apiUrl
    */
-  constructor(sdkId, pem) {
+  constructor(sdkId, pem, { apiUrl } = {}) {
     this.sdkId = sdkId;
     this.pem = pem;
 
+    const options = {
+      apiUrl: apiUrl || config.yoti.connectApi,
+    };
+
+    this.amlService = new AmlService(sdkId, pem, options);
+    this.profileService = new ProfileService(sdkId, pem, options);
+    this.dynamicShareService = new DynamicShareService(sdkId, pem, options);
+
     /** @deprecated replaced by this.sdkId */
```

Comment:
Can this `@deprecated` be removed and the line that follows too?

Reply:
Well, this probably been there for some time...
I would like to remove it indeed, but we shall do that in as part of a major release with breaking changes, wouldn't it be better?